flashcards/backend/players/models.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+
+import pytz
+import uuid
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from config import settings
+
+
+class AccountType(models.Model):
```

Comment:
I would pass on AccuntType for now, does not really correspond to the story your task is related to.